Regression: Building with `REBUILD_WEB` broken since merge of #7209

### What is the issue?

```bash
cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DREBUILD_WEB=ON
```
```
[ 29%] Generating public_html/transmission-app.css, public_html/transmission-app.css.LEGAL.txt, public_html/transmission-app.css.map, public_html/transmission-app.js, public_html/transmission-app.js.LEGAL.txt, public_html/transmission-app.js.map
Error copying file "/tmp/transmission/web/prettier.config.js" to "/tmp/transmission/build/web".
Error copying file "/tmp/transmission/web/stylelint.config.js" to "/tmp/transmission/build/web".
gmake[2]: *** [web/CMakeFiles/transmission-web.dir/build.make:124: web/public_html/transmission-app.css] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:598: web/CMakeFiles/transmission-web.dir/all] Error 2
gmake: *** [Makefile:166: all] Error 2
```
This failure is not encountered with the `main` commit preceding #7209 merge: [`dce515f37cdbd92b7b12158c82add7382e053909`](https://github.com/transmission/transmission/commit/dce515f37cdbd92b7b12158c82add7382e053909)

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

f0e7c3ed748971d0392c118c2799c22880d6cd81

--- a/web/CMakeLists.txt
+++ b/web/CMakeLists.txt
@@ -65,8 +65,8 @@ add_custom_command(
         "${CMAKE_CURRENT_SOURCE_DIR}/esbuild.mjs"
         "${CMAKE_CURRENT_SOURCE_DIR}/package-lock.json"
         "${CMAKE_CURRENT_SOURCE_DIR}/package.json"
-        "${CMAKE_CURRENT_SOURCE_DIR}/prettier.config.js"
-        "${CMAKE_CURRENT_SOURCE_DIR}/stylelint.config.js"
+        "${CMAKE_CURRENT_SOURCE_DIR}/prettier.config.cjs"
+        "${CMAKE_CURRENT_SOURCE_DIR}/stylelint.config.cjs"
         "${CMAKE_CURRENT_BINARY_DIR}"
     COMMAND "${CMAKE_COMMAND}" -E copy_directory
         "${CMAKE_CURRENT_SOURCE_DIR}/assets" "assets"

